spec/fixtures/proto-google-common-protos-0.1.9.pom

```diff
@@ -53,3 +53,4 @@
     </dependency>
   </dependencies>
 </project>
+<publishedAt>Fri, 11 Oct 2019 20:00:00 GMT</publishedAt>
```

Comment:
From what I can tell this was stubbed out in the rest of the specs and doesn't show up on the POM files on maven (e.g. https://search.maven.org/artifact/org.jacoco/org.jacoco.core/0.8.7/pom and https://search.maven.org/artifact/org.jacoco/org.jacoco.build/0.8.7/pom), but it does show up in the xml we pull in.